cmd/daemon/kubernetes/pods.go

```diff
@@ -0,0 +1,207 @@
+package kubernetes
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"math"
+	"strings"
+
+	"github.com/lunarway/release-manager/internal/http"
+	"github.com/lunarway/release-manager/internal/log"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func (c *Client) HandlePodErrors(ctx context.Context) error {
+	// TODO; See if it's possible to use FieldSelector to limit events received.
+	watcher, err := c.clientset.CoreV1().Pods("").Watch(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for {
+		select {
+		case <-ctx.Done():
+			watcher.Stop()
+		case e, ok := <-watcher.ResultChan():
+			if !ok {
+				return ErrWatcherClosed
+			}
+			if e.Object == nil {
+				continue
+			}
+			pod, ok := e.Object.(*corev1.Pod)
+			if !ok {
+				continue
+			}
+			// Is this a pod managed by the release manager - and does it contain the information needed?
+			if !isPodCorrectlyAnnotated(pod) {
+				continue
+			}
+			// Avoid reporting on pods that has been marked for termination
+			if isPodMarkedForTermination(pod) {
+				continue
+			}
+
+			if isPodInCrashLoopBackOff(pod) {
+				log.Infof("Pod: %s is in CrashLoopBackOff", pod.Name)
+				restartCount := pod.Status.ContainerStatuses[0].RestartCount
+				if math.Mod(float64(restartCount), c.moduloCrashReportNotif) != 1 {
+					continue
+				}
+				var errorContainers []http.ContainerError
+				for _, cst := range pod.Status.ContainerStatuses {
+					if cst.State.Waiting != nil && cst.State.Waiting.Reason == "CrashLoopBackOff" {
+
+						logs, err := getLogs(ctx, c.clientset, pod.Name, cst.Name, pod.Namespace)
+						if err != nil {
+							log.Errorf("Error retrieving logs from pod: %s, container: %s, namespace: %s", pod.Name, cst.Name, pod.Namespace)
+						}
+
+						errorContainers = append(errorContainers, http.ContainerError{
+							Name:         cst.Name,
+							ErrorMessage: logs,
+							Type:         "CrashLoopBackOff",
+						})
+					}
+				}
+
+				err = c.exporter.SendPodErrorEvent(ctx, http.PodErrorEvent{
+					PodName:     pod.Name,
+					Namespace:   pod.Namespace,
+					Errors:      errorContainers,
+					ArtifactID:  pod.Annotations["lunarway.com/artifact-id"],
+					AuthorEmail: pod.Annotations["lunarway.com/author"],
+				})
+				if err != nil {
+					log.Errorf("error SendCrashLoopBackOffEvent")
```

Comment:
The error is missing from this log.
```suggestion
					log.Errorf("Failed to send crash loop backoff event: %v", err)
```